source/scss/1200.scss

```diff
@@ -1,15 +1,39 @@
-.menu {
-    &.main {
-        li {
-            a {
-                font-size: 2em;
+header {
+
+  .nav-main {
+   
+    nav {
+   
+      li {
+   
+        a {
+          font-size: 2em !important;
```

Comment:
Why is there `!important` here, what lines of code is it overriding?

Reply:
doesn't seem to be respecting the break point unless I use the important

Reply:
Fixed now